src/stdlib/SDL_mslibc.c

```diff
@@ -696,8 +696,80 @@ void __declspec(naked) _aullshr()
     /* *INDENT-ON* */
 }
 
+void __declspec(naked) _chkstk(void)
+{
+    __asm {
+        push        ecx
+        mov         ecx,esp     ; lea         ecx,dword ptr [esp]+4
+        add         ecx,4
```

Comment:
Clang should be fine with lea, just need to put the offset inside the brackets:
`lea ecx,dword ptr [esp+4]`

Reply:
That seems to make clang-cl happy.
Thanks!